build.sbt

```diff
@@ -92,6 +93,7 @@ lazy val core = project
       "org.scalatest"              %% "scalatest"                      % scalaTestVersion           % Test,
       "org.scalatestplus"          %% "scalacheck-1-15"                % scalaTestScalaCheckVersion % Test,
       "org.mdedetrich"             %% "scalacheck"                     % scalaCheckVersion          % Test,
+      "com.rallyhealth"            %% "scalacheck-ops_1-15"            % scalaCheckOpsVersion       % Test,
```

Comment:
This is an extension library for scalacheck that adds generators for different types. Specifically we needed a generator for `OffsetDateTime`